src/lib/Runner.ts

```diff
@@ -333,6 +333,21 @@ export class Runner {
     return true;
   };
 
+  moveRequestToTopOfQueue = async (requestId: string, user: ISessionUser): Promise<boolean> => {
+    const landRequestStatus = await this.queue.maybeGetStatusForQueuedRequestById(requestId);
+    if (!landRequestStatus) return false;
+
+    const requestAtTop = (await this.getQueue())[0];
```

Comment:
Hmmm.. What happens if we call this and theres nothing there?
I guess it could blow up, but wouldn't really break anything?

Might be nice to guard either way though

Reply:
True, will push quick fix